### PR TITLE
Address simple TODOs in the code

### PR DESCRIPTION
**Pull request by drummyfish**
I am getting familiar with the code and I fixed some TODOs along the way. I know I should separate different issues into multiple PRs, but these were trivial ones so I thought I'd do it in one go (will separate them if you insist).

- Getting into taxi gives you $25, tested. (~~Though I think each taxi should only give the money once, should I add a check?~~ - did this)
- Removed redundant condition, tested.
- Added some documenting comment after inspecting DFF models with Blender.

### CODE REVIEW

**Comment by ShFil119** [comment]
What do you mean UV?

**Comment by drummyfish** [comment]
UV mapping of vertices for texturing, should I go more in depth to find out where exactly A vs B is used in game?

**Comment by drummyfish** [comment]
reopening to force test re-run

**Comment by ShFil119** [comment]
1) Reopening is making the log quite awful to read.
2) You can restart test with forceful push. Or...
3) You can set travis to test pulls on your own fork.

**Comment by drummyfish** [comment]
Thanks, next time I'll know :) I passed now.

File: rwengine/src/objects/VehicleObject.cpp [hunk 1]
```diff
@@ -792,7 +790,10 @@ void VehicleObject::grantOccupantRewards(CharacterObject* character) {
             || getVehicle()->vehiclename_ == "CABBIE"
             || getVehicle()->vehiclename_ == "BORGNINE") {
             // Earn $25 from taxi cabs
-            /// @todo implement this
+            if (!moneyCollected) {
```
Comment (danhedron) [inline rwengine/src/objects/VehicleObject.cpp]: The collectSpecial() function already checks if the vehicle occupant reward has been granted. The ``moneyCollected`` variable is redundant.

**Comment by JayFoxRox** [comment]
Please squash up, 5 commits for +2 -6 is overkill.

Otherwise LGTM (Code reviewed, did not test).

**Comment by darkf** [comment]
LGTM.

**Pull request merged**